augur/export_v2.py

```diff
@@ -899,10 +888,10 @@ def register_parser(parent_subparsers):
     config = parser.add_argument_group(
         title="DISPLAY CONFIGURATION",
         description="These control the display settings for auspice. \
-            You can supply a config JSON (which has all available options) or command line arguments (which are more limited but great to get started). \
+            You can supply one or more config JSONs (which have all available options) or command line arguments (which are more limited but great to get started). \
             Supplying both is fine too, command line args will overrule what is set in the config file!"
     )
-    config.add_argument('--auspice-config', metavar="JSON", help="Auspice configuration file")
+    config.add_argument('--auspice-config', metavar="JSON", nargs='+', action=ExtendOverwriteDefault, default=[], help="Auspice configuration file(s)")
```

Comment:
I think it'd be helpful to explicitly state in the help message that if multiple config files are provided then they would be merged and give a brief description of how the values are merged.

Reply:
I don't think I can convey _how_ the merging behaviour works within the `--help` message, but I can totally say they'll be merged. Separately I plan to write a docs page, but I was going to leave it until it's been in use for a while internally so that it's easier to make breaking changes, but I can add them now if you think that's preferable?

Reply:
[Updated the argument group's help message](https://github.com/nextstrain/augur/compare/23595ee4deb4eebf66d73272f20d54dbfc420f03..b294cbcdb89cef2c2ea4ae7e5a3f8957a67abb5d)

Reply:
The updated help message is fine for now! I just know there will eventually be questions on how the auspice configs are merged once this is more widely used.